app.js

```diff
@@ -1,51 +1,60 @@
 'use strict';
 
-var storeOne = {
-  name:'First and Pike',
-  minCust:'23',
-  maxCust:'65',
-  avgSale:'6.3'
-};
+var shops = [];
+var totals = [];
+console.log(totals);
+var storeHours = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13];
 
-var storeTwo = {
-  name:'Seatac Airport',
-  minCust:'3',
-  maxCust:'24',
-  avgSale:'1.2'
-};
+function Store(name, minCust, maxCust, avgSale){
+  this.name = name;
+  this.minCust = minCust;
+  this.maxCust = maxCust;
+  this.avgSale = avgSale;
+  this.salesPerHour = [];
+  shops.push(this);
 
-var storeThree = {
-  name:'Seattle Center',
-  minCust:'11',
-  maxCust:'38',
-  avgSale:'3.7'
-};
+  this.cookiesPerHour = function(){
+    for (var i = 0; i < storeHours.length; i++){
+      var customerPerHour = Math.floor(Math.random() * (this.maxCust - this.minCust) + this.minCust);
+      var cookiesPerHour = Math.floor(customerPerHour * this.avgSale);
+      this.salesPerHour.push(cookiesPerHour);
+    }
+  };
+  this.totalPerDay = function(){
+    this.cookiesPerHour();
+    var total = 0;
+    for (var c = 0; c < this.salesPerHour.length; c++){
+      total += this.salesPerHour[c];
+    }
+    this.salesPerHour.push(total);
+  };
+}
 
-var storeFour = {
-  name:'Cap Hill',
-  minCust:'20',
-  maxCust:'38',
-  avgSale:'2.3'
-};
+new Store('First and Pike', 23, 65, 6.3);
+new Store('Seatac Airport', 3, 24, 1.2);
+new Store('Seattle Center', 11, 38, 3.7);
+new Store('Cap Hill', 20, 38, 2.3);
+new Store('Alki', 2, 16, 4.6);
 
-var storeFive = {
-  name:'Alki',
-  minCust:'2',
-  maxCust:'16',
-  avgSale:'4.6'
-};
+for (var k = 0; k < shops.length; k++){
+  shops[k].totalPerDay();
+}
 
-var salesPerHour = [];
-console.log(salesPerHour);
-var totalSalesPerHour = [];
-var totalSales = [];
+var runIt = function(){
+  for (var m = 0; m < shops[0].salesPerHour.length; m++){
+    var allCookies = 0;
+    for (var w = 0; w < shops.length; w++){
+      allCookies += shops[w].salesPerHour[m];
+    }
+    totals.push(allCookies);
+  }
+};
+runIt();
 
 var body = document.getElementsByTagName('body')[0];
 var newtable = document.createElement('table');
-var shop = [storeOne, storeTwo, storeThree, storeFour, storeFive];
-for (var i = 0; i < shop.lenght; i++);
+newtable.className = 'table';
 body.appendChild(newtable);
-
 for (var t = -1; t < 15; t++){
```

Comment:
Why is this for loop just sitting around in the global scope? put this in a function!